data/mods/Magiclysm/items/enchanted.json

```diff
@@ -20,25 +20,66 @@
     "type": "GENERIC",
     "symbol": "/",
     "color": "brown",
+    "name": { "str": "minor staff of the magi", "str_pl": "minor staves of the magi" },
+    "description": "A simple carved staff, made of wood.  It faintly glows with magic when you cast spells. Wielding it slightly increases your mana regeneration",
+    "price": 10000,
+    "material": [ "wood" ],
+    "techniques": [ "WBLOCK_2", "RAPID", "SWEEP" ],
+    "flags": [ "SHEATH_SPEAR", "ALWAYS_TWOHAND", "DURABLE_MELEE", "MAGIC_FOCUS" ],
+    "relic_data": {
+      "passive_effects": [ { "has": "WIELD", "condition": "ALWAYS", "values": [ { "value": "REGEN_MANA", "multiply": 0.5 } ] } ]
+    },
+    "weight": "1400 g",
+    "volume": "3 L",
+    "bashing": 17,
+    "category": "weapons",
+    "to_hit": 3
+  },
+  {
+    "id": "magi_staff_lesser",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "brown",
     "name": { "str": "lesser staff of the magi", "str_pl": "lesser staves of the magi" },
-    "description": "A beautifully carved staff, made of enchanted wood and mithril.  It faintly glows with magic when you cast spells, but it is not a sturdy melee weapon.",
+    "description": "A beautifully carved staff, made of enchanted wood.  It faintly glows with magic when you cast spells. Wielding it increases your mana regeneration",
     "price": 40000,
     "material": [ "wood" ],
-    "flags": [ "SHEATH_SPEAR", "ALWAYS_TWOHAND", "FRAGILE_MELEE", "MAGIC_FOCUS" ],
+    "techniques": [ "WBLOCK_2", "RAPID", "SWEEP" ],
+    "flags": [ "SHEATH_SPEAR", "ALWAYS_TWOHAND", "DURABLE_MELEE", "MAGIC_FOCUS" ],
+    "relic_data": {
+      "passive_effects": [ { "has": "WIELD", "condition": "ALWAYS", "values": [ { "value": "REGEN_MANA", "multiply": 1.25 } ] } ]
+    },
+    "weight": "1400 g",
+    "volume": "3 L",
+    "bashing": 17,
+    "category": "weapons",
+    "to_hit": 3
+  },
+  {
+    "id": "magi_staff_greater",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "brown",
+    "name": { "str": "greater staff of the magi", "str_pl": "greater staves of the magi" },
+    "description": "A beautifully carved staff, made of enchanted wood and mithril.  It faintly glows with magic when you cast spells. Wielding it greatly increases your mana regeneration",
+    "price": 1000000,
+    "material": [ "wood" ],
```

Comment:
```suggestion
    "material": [ "wood", "mithril_metal" ],
```
It does make sense for this one to be more durable, since it's partially mithril.